modules/appeals_api/app/models/appeals_api/higher_level_review.rb

```diff
@@ -20,11 +20,11 @@ class HigherLevelReview < ApplicationRecord
     }
 
     scope :v1, lambda {
-      where(api_version: 'V1')
+      where(api_version: 'v1')
```

Comment:
Yup. Unless we go back through our old records and update them to be downcase, this scope will only return records of HLRs created after this PR gets deployed.

...which for v1 will be 0 records, since no HLRv1s should ever be created after its removal from our codebase a few weeks ago.

Reply:
no problem. I think I realized halfway through fixing tests that this was the more disruptive change, so this'll end up being a much smaller PR after reverting that change/updating a handful of downcase strings to be upcase

Reply:
the main thing I'm remembering now is all the schema version stuff is downcase, since some of it ultimately gets used to resolve some `v2` file paths as well. I don't think it's a big deal to have `api_version`s to be consistently upcase and `schema_versions` consistently downcased, but also fine changing the schema version strings to upcase if that's preferable